bundles/org.openhab.binding.ekey/src/main/java/org/openhab/binding/ekey/internal/api/EkeyUdpPacketReceiver.java

```diff
@@ -48,10 +48,10 @@ public class EkeyUdpPacketReceiver {
 
     private boolean connected = false;
 
-    public EkeyUdpPacketReceiver(final String ipAddress, final int port, final String readerThreadName) {
+    public EkeyUdpPacketReceiver(final String ipAddress, final int port, final String thingUID) {
```

Comment:
I think this was better before. It seems more natural to provide `readerThreadName` explicitly. Having to provide thingUID instead makes you think: "What does the UDP packet receiver need the Thing UID for?"

Reply:
Changed.